public/app/core/components/Breadcrumb/BreadcrumbSections.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react';
+import { cx } from 'emotion';
+import { useSelector } from 'react-redux';
+import { useStyles } from '@grafana/ui';
+import { StoreState } from 'app/types';
+import { getStyles } from './Breadcrumb.styles';
+import { BreadcrumbProps } from './Breadcrumb.types';
+
+export const BreadcrumbSections: FC<BreadcrumbProps> = ({ pageModel }) => {
+  const styles = useStyles(getStyles);
+  const locationPath = useSelector((state: StoreState) => state.location.path);
+  const currentLocation = locationPath.slice(1);
+  const { title, path: modelPath, children } = pageModel;
+
+  const isCurrentPage = currentLocation === modelPath;
+  const activeChild = children?.find(
+    child => currentLocation.startsWith(`${child.path}/`) || currentLocation === child.path
+  );
+
+  return (
+    <span data-qa="breadcrumb-section" className={cx(isCurrentPage && styles.currentPage)}>
+      {isCurrentPage ? (
+        title
+      ) : (
+        <a data-qa="breadcrumb-section-link" className={styles.link} href={modelPath}>
```

Comment:
Maybe a `Link` instead? Otherwise this will cause a full page refresh

Reply:
> Maybe a `Link` instead? Otherwise this will cause a full page refresh

Maybe not, but another question is: would we ever need to integrate it with the router? Even if not, what if we need to apply the `active` class, which `Link` provides out of the box? Something to think about.

Reply:
We're not using `react-router` here and not even the `active` class, I don't think it might be useful here. The "active" element is always the last one and it's not a link :)

Reply:
roger 👍